### DISCUSSION TITLE
How to insert a single blank page for use with some mis-aligned PDF ?

### DISCUSSION THREAD

**Discussion by vacarogit** [sumatrapdfreader/sumatrapdf#2928] [Ideas]
especially in two-column face to face mode  (providing a toggle switch for that is even better)

Thank you for considering.

**Comment by GitHubRulesOK** [comment]
Hmm that is an editing function [I discussed elsewhere this week](https://stackoverflow.com/questions/62199136/merge-pdf-files-with-extra-blank-page-at-the-end-of-odd-paged-documents-qpdf/73575464#73575464), It is not as easy as it seems so powerhouse `qpdf` pdf manipulation currently has no simple means however its trivial with cpdf `cpdf -pad-multiple 2 in.pdf -o out.pdf` and perhaps might be a simpler means in mutool but not tested.

The big issue is injecting a page upsets all internal links unless its the last page.

for flipping order its easy see manga mode and that can be a hotkey see https://github.com/sumatrapdfreader/sumatrapdf/discussions/2915

**Reply by vacarogit** [reply]
I found the "book view" mode is doing a job for the issue. Turning book view will insert a virtual green page as page 000, so any oddly aligned book shifted to good alignment afterwards. I need a toggle in context for fast access~ haha
